br/pkg/backup/client.go

```diff
@@ -680,6 +680,10 @@ func skipUnsupportedDDLJob(job *model.Job) bool {
 		model.ActionAlterTableAttributes,
 		model.ActionAlterTablePartitionAttributes:
 		return true
+	case model.ActionCreateResourceGroup,
```

Comment:
I think it's better to let br support resource group DDL. Is there any problem to do so?

Reply:
They are not the same. Because placement policy is relate to the cluster topology, so restore the placement rule to a different cluster may be not very useful. But resource group is not, I think it's useful to just restore the resource group settings. BTW, if we want to not ignore the resource group DDL, we should also skip the user bind resource group name attribute to let it in a consistent state, it seems hard to do so.

Reply:
I skip this check first, and will do more after this PR. record in https://github.com/pingcap/tidb/issues/40371.